util.go

```diff
@@ -23,6 +23,12 @@ func (n NodeID) Bytes() []byte {
 	return bytes
 }
 
+// BroadcastNodeID returns the special NodeID used when broadcasting a packet to a channel.
+func BroadcastNodeID() NodeID {
```

Comment:
I think I'd rather see this as `const BroadcastNodeID NodeID = math.MaxUint32` unless you have an argument for your way?

Reply:
or just use `4294967295`  instead of importing `math`

Reply:
I don't think there's any reason to avoid importing math tbh - it's clearer what the significance of the value is. I can switch this to a const.